docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -63,6 +56,20 @@ $ sudo usermod -a -G libvirtd <username>
 $ newgrp libvirtd
 ----
 
+. As root, install the KVM driver binary and make it executable as follows (For Ubuntu-16.04 and higher):
```

Comment:
We should not mention `For Ubuntu-16.04 and higher` as this is generic for all distribution. Also we have the specific section for Ubuntu

Reply:
s/KVM driver binary/ KVM docker machine plugin/

Reply:
> We should not mention For Ubuntu-16.04 and higher as this is generic for all distributions.

@LalatenduMohanty I don't understand that, what do you mean this is generic for all distributions?

Reply:
@praveenkumar This is common for Debian as well as Ubuntu .Also you have a separate section for Ubuntu anyway. So we do not need to mention Ubuntu specific texts here. You can mention those in the Ubuntu section .

Reply:
@LalatenduMohanty I still don't get, here I mentioning ubuntu because driver have different version for different Ubuntu version, what I am missing here.

Reply:
I got confused by the way instruction for Ubuntu has been written. I will resolve this comments and write new one